go.mod

```diff
@@ -20,9 +20,9 @@ require (
 	github.com/pmezard/go-difflib v1.0.0
 	github.com/prometheus/client_golang v1.10.0
 	github.com/sirupsen/logrus v1.8.1
-	github.com/spf13/cobra v0.0.5
+	github.com/spf13/cobra v1.3.0
 	github.com/spf13/pflag v1.0.5
-	github.com/spf13/viper v1.7.1
+	github.com/spf13/viper v1.10.0
```

Comment:
Anything good in the new versions?

Reply:
None that I looked at, this as done as part of the `go mod tidy` cleanup I had to do to make this run.